Gemfile

```diff
@@ -20,4 +20,5 @@ gem "test-unit"
 
 group :test, :development do
   gem "pry"
+  gem "ruby-prof"
```

Comment:
What does ruby-prof do? Also, is it worth pinning our dependencies for the func tests like we do elsewhere?

Reply:
This goes get locked to 0.17 in `Gemfile.lock`

Reply:
I mean in the Gemfile, so that we get a consistent lock file. Currently, I think, if someone runs bundle it'll update the gem version and stuff could creep up?

Reply:
Ah righto. Seems sensible. 👍

Reply:
Pinned!